### DISCUSSION TITLE
Authentication to configuration content

### DISCUSSION THREAD

**Discussion by a8ree** [Azure/GuestConfiguration#299] [General]
We have a standard that we don't allow public / unauthenticated access to our resources. I'm trying to determine how the VMs that will be managed using Guest Configuration will access the ContentUri

Is it possible for Guest Configuration to leverage System or User assigned identities to pull Content?

**Comment by MutemwaRMasheke** [comment]
Hello! @a8ree your question is spot on. We are nearly done testing out an implementation of this exact feature and will be in touch soon when we are clear on an ETA. Thanks!

**Reply by a8ree** [reply]
Great @MutemwaRMasheke. I will look forward to that feature.
In the meantime can I use SAS keys of does the content need to be public?

**Reply by MutemwaRMasheke** [reply]
You can certainly use a SAS token by following the instructions here: https://learn.microsoft.com/en-us/azure/governance/machine-configuration/how-to/develop-custom-package/4-publish-package

**Comment by a8ree** [comment]
Thanks! Got it
